tasks/Graph Coloring Task

```diff
@@ -0,0 +1,54 @@
+# Summary
+> This study looked at the influence of structural properties on naturally occurring networks given their influence on behavior and dynamics. Through the use of 
+network coloring problems subjects differentiated their color from neighbors and those generated by preferential attachment made solving the coloring problem more 
+difficult than did networks based on cyclical structures, and "small worlds" networks were easier still. This indicates providing more information can have 
+opposite effects on performance, depending on network structure. 
+
+
+# References
+> N/A
```

Comment:
https://science.sciencemag.org/content/sci/suppl/2006/08/08/313.5788.824.DC1/Kearns_SOM.pdf
This is the supporting material they reference in number 18 of the reference list!